index.js

```diff
@@ -103,3 +110,35 @@ const bookInput = () => {
 };
 
 addBtn.addEventListener('click', bookInput);
```

Comment:
[Optional]
It will be better if you change the page to the book list after adding a new book.

Reply:
Done